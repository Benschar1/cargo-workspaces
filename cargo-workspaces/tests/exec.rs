mod utils;
#[cfg(not(windows))]
use insta::assert_snapshot;

// #[cfg(windows)]
// static PRINT: &str = "cd";

#[cfg(not(windows))]
static PRINT: &str = "cat";

// TODO: Get exec test working on windows
#[cfg(not(windows))]
#[test]
fn test_normal() {
    let (out, err) = utils::run("../fixtures/normal", &["ws", "exec", PRINT, "Cargo.toml"]);
    assert_snapshot!(err);
    assert_snapshot!(out);
}

// TODO: Get exec test working on windows
#[cfg(not(windows))]
#[test]
fn test_normal_ignore() {
    let (out, err) = utils::run(
        "../fixtures/normal",
        &["ws", "exec", "--ignore={dep2,top}", PRINT, "Cargo.toml"],
    );
    assert_snapshot!(err);
    assert_snapshot!(out);
}
