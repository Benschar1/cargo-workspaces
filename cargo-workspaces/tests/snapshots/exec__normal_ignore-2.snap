---
source: tests/exec.rs
assertion_line: 29
expression: out

---
[package]
name = "dep1"
version = "0.1.0"
authors = ["Pavan Kumar Sunkara <pavan.sss1991@gmail.com>"]
edition = "2018"

[dependencies]

